src/index.js

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import { Provider } from 'react-redux';
+/* import { BrowserRouter as Router } from 'react-router-dom'; */
 import App from './App';
-import store from './app/store';
 import * as serviceWorker from './serviceWorker';
+import 'tw-elements';
```

Comment:
Can you tidy this up a bit, move the global imports to top and leave a blank line then add the local imports.

Reply:
- I organized the code as you mentioned in index.js
- Navbar component is in the in src/components/Navbar/ with LanguageDropdown component as part of the navbar.js
- Buttons have been revised and the transition has been added/edited.
- Responsive issue of navbar revised from scratch. Now, it's working as it should. Also, I viewed a similar website to understand and implement hamburger menu for mobile. Since our project includes many sections and buttons, I thought it would be more appropriate to switch to the hamburger menu after 767 px. I have also included responsive images of another website that has a similar purpose.
- I tried to put our logo on the left side, but It didn't happen because I couldn't change the location of the navbar. So I put this logo on the right side. Also, when the hamburger menu is opened, the logo appears, otherwise, it remains hidden. Since I couldn't fix it, I implemented it this way. I'm open to any advice or guidance.

Reply:
_**Video and images about my work are below (I decreased the size of the logo for mobile after the video)**_
<br>

https://user-images.githubusercontent.com/60944453/163273244-1fb4ab74-bfcb-442b-bf02-382bdd014d0e.mp4

_Images for Mobile and Desktop_

![3](https://user-images.githubusercontent.com/60944453/163274142-cbda8926-bf6c-4560-91c8-785c75f2a2f4.PNG)

![4](https://user-images.githubusercontent.com/60944453/163272903-a72ebccc-1566-4e59-bdd6-18ac5c1631cf.png)

_Responsive images of another website that has a similar purpose._
<br>
![ornek](https://user-images.githubusercontent.com/60944453/163273052-d43347dd-d87b-41ad-9f66-8b11d3d6d7ae.png)

Reply:
It looks really cool. Great work. 👍🏼

Reply:
I just checked your Node.js CI / builds(12.x and 14.x), don't forget to add ` ; ` to last line of postcss.config.js.

Reply:
Thank you Teoman, your suggestion helped me to solve this error.